configs/eth_odin_v4.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I don't think Odin needs a separate ethernet config, it can re-use the generic eth config?